tinygrad/renderer/__init__.py

```diff
@@ -9,19 +9,15 @@
 @dataclass(frozen=True)
 class TensorCore: # D = A * B + C, A is (M x K), B is (K x N), C and D are (M x N)
   dims: tuple[int,int,int] # N, M, K
+  upcast_size: tuple[int, int, int] # A, B, C/D
```

Comment:
Can you explain what this is in a comment?

Reply:
Yes